### DISCUSSION TITLE
Milvus Standalone , Minio S3 Connection

### DISCUSSION THREAD

**Discussion by obadaqasem** [milvus-io/milvus#34850] [Q&A and General discussion]
Dear Milvus Team, 

I would like to thank you for this amazing artwork first of all, I have a little question, I am now in my publish stage, and I am trying to connect minio with the s3, I am using milvus standalone v2.3, how can I add and configure minio through the docker-compose file . Thank you so much.

**Comment by xiaofan-luan** [comment]
quick question, do you want to use S3 or Minio as storage?

I don't quite understand "connect minio with the s3"

**Comment by xiaofan-luan** [comment]
There is actaully one document talks about how to depoly milvus on aws. We directly use S3 instead of minio

**Comment by obadaqasem** [comment]
I want s3 for the storage

**Reply by xiaofan-luan** [reply]
check https://milvus.io/docs/eks.md

**Comment by a28124791** [comment]
可不可以连Ozone？

**Reply by xiaofan-luan** [reply]
> 可不可以连Ozone？

目前还不支持，如果arrow cpp能够连的话应该比较容易做

**Reply by a28124791** [reply]
Ozone不是也支持s3吗？不行吗？

**Reply by xiaofan-luan** [reply]
如果是S3协议我们都可以支持，只要S3 cpp客户端能访问即可
